compiler/luci-interpreter/include/luci_interpreter/core/Tensor.h

```diff
@@ -118,15 +115,16 @@ class Tensor
 
   template <typename T> const T *data() const
   {
-    assert(_data_allocated);
-    return reinterpret_cast<const T *>(_data.get());
+    static_assert(std::is_same<uint8_t, char>::value or
```

Comment:
@SlavikMIPT 
If I recall it correctly, you wanted this check.
Q) Isn't it enough to check if `uint8_t == unsigned char` here?